core/src/env.rs

```diff
@@ -275,15 +275,15 @@ mod tests {
     #[test]
     /// Test that the new LaterExt object contains reference on None value
     fn empty_ext_creation() {
-        let ext = LaterExt::<ExtImplementedStruct>::new();
+        let ext = <LaterExt<ExtImplementedStruct> as Default>::default();
 
         assert_eq!(ext.inner, Rc::new(RefCell::new(None)));
     }
 
     #[test]
     /// Test that we are able to set and unset LaterExt value
     fn setting_and_unsetting_inner_ext() {
-        let mut ext = LaterExt::<ExtImplementedStruct>::new();
+        let mut ext = <LaterExt<ExtImplementedStruct> as Default>::default();
```

Comment:
We don't need to specify the type if it can be inferred.

```suggestion
        let mut ext: LaterExt<_> = Default::default();
```